tests/foreman/api/test_activationkey.py

```diff
@@ -146,6 +164,27 @@ def test_positive_update_1(self):
                 actual = {attr: getattr(act_key, attr) for attr in want.keys()}
                 self.assertEqual(want, actual)
 
+    def test_positive_update_2(self):
+        """@Test: Create activation key providing the initial name, then update
+        its name to another valid name.
+
+        @Assert: Activation key is created, and its name can be updated.
+
+        @Feature: ActivationKey
+
```

Comment:
No need to add this blank line.

For more info check: https://www.python.org/dev/peps/pep-0008/#documentation-strings

PS: I know that we have been doing this for a long time, but some time ago I read again that section and figured out that we ensured a non PEP8 standard.